### ISSUE TITLE
Expose stats to let user know if there's any error during the compilation

### ISSUE THREAD

**Issue by egoist** [webpack/webpack-dev-middleware#126]
https://github.com/webpack/webpack-dev-middleware/blob/master/middleware.js#L89

```js
// update to:
cb(stats);

// user could check errors with:
devMiddleWare.waitUntilValid(stats => {
  if (stats.hasError()) {} // blah blah...
})
```

**Comment by SpaceK33z** [comment]
PR welcome.

**Comment by SpaceK33z** [comment]
Fixed in #127.
